flow/e2e/clickhouse/peer_flow_ch_test.go

```diff
@@ -263,3 +263,81 @@ func (s ClickHouseSuite) Test_Date32() {
 	env.Cancel()
 	e2e.RequireEnvCanceled(s.t, env)
 }
+
+func (s ClickHouseSuite) Test_Schema_Changes_CH() {
```

Comment:
If CH implements `connectors.GetTableSchemaConnector` then generic schema changes test will be activated for CH